examples/hello-numpy-cross-val/config/config_fed_server.json

```diff
@@ -32,6 +32,11 @@
       "id": "formatter",
       "path": "nvflare.app_common.np.np_formatter.NPFormatter",
       "args": {}
+    },
+    {
```

Comment:
is this something new ?

Reply:
no, this is not new stuff, it was specified in the text of the documentation, but somehow is not included in the example config.